zerver/lib/import_realm.py

```diff
@@ -584,14 +584,16 @@ def bulk_import_client(data: TableData, model: Any, table: TableName) -> None:
             client = Client.objects.create(name=item['name'])
         update_id_map(table='client', old_id=item['id'], new_id=client.id)
 
-def import_uploads(import_dir: Path, processes: int, processing_avatars: bool=False,
-                   processing_emojis: bool=False) -> None:
+def import_uploads(realm: Realm, import_dir: Path, processes: int, processing_avatars: bool=False,
+                   processing_emojis: bool=False, processing_realm_icons: bool=False) -> None:
     if processing_avatars and processing_emojis:
         raise AssertionError("Cannot import avatars and emojis at the same time!")
     if processing_avatars:
         logging.info("Importing avatars")
     elif processing_emojis:
         logging.info("Importing emojis")
+    elif processing_realm_icons:
+        logging.info("Importing realm icons and logos")
     else:
         logging.info("Importing uploaded files")
```

Comment:
These blocks might be better refactored with a single `upload_type="emoji"` argument rather than having a lot of these complex if/else statements.  Definitely a follow-up commit to do, but I think it'd be better now that we have 4 of these :)

Reply:
Or maybe better, we could define a class UploadType with fields like `upload_type.has_user_profile_id` and `upload_type.bucket_name` that define the answers to all of these random-feeling if statements.